### PR TITLE
fix(dashboards): use rate_interval instead of interval

### LINKED ISSUE #274: Panels in the alertmanager dashboard are emtpy
### Bug Description

Lots of panels in the dashboard show "no data" due to the use of `$__interval` instead of `$__rate_interval`.

On a sidenote, the **Notification Durations** show time series that are named `0.99q`, but it might not be clear to a customer what that means (or what quantiles are). We should try to make that panel more immediate to understand, maybe adding a tooltip to explain it.

### To Reproduce

-

### Environment

-

### Relevant log output

```shell
-
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lucabello**
## Issue
Closes #274.

## Solution
Use `$__rate_interval` instead of `$__interval`.

### CODE REVIEW

**Comment by mmkay** [comment]
One thing to potentially consider here: https://hoelz.ro/blog/use-caution-when-using-rate_interval-along-with-increase

This might change the values scale visible in the dashboards. Might be surprising to some?

**Comment by lucabello** [comment]
You're right, that replace was too lazy :) Now we only changed to `$__rate_interval` in `rate()` queries.

File: src/grafana_dashboards/alertmanager_rev5.json.tmpl [hunk 1]
```diff
@@ -6793,7 +6793,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__interval])",
+              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval])",
```
Comment (mmkay) [inline src/grafana_dashboards/alertmanager_rev5.json.tmpl:6796]:
```suggestion
              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval])",
```

File: src/grafana_dashboards/alertmanager_rev5.json.tmpl [hunk 2]
```diff
@@ -6895,7 +6895,7 @@
...
```
Comment (mmkay) [inline src/grafana_dashboards/alertmanager_rev5.json.tmpl:6898]:
```suggestion
              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval])",
```

File: src/grafana_dashboards/alertmanager_rev5.json.tmpl [hunk 3]
```diff
@@ -6997,7 +6997,7 @@
...
```
Comment (mmkay) [inline src/grafana_dashboards/alertmanager_rev5.json.tmpl:7000]:
```suggestion
              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval])",
```

File: src/grafana_dashboards/alertmanager_rev5.json.tmpl [hunk 4]
```diff
@@ -7099,7 +7099,7 @@
...
```
Comment (mmkay) [inline src/grafana_dashboards/alertmanager_rev5.json.tmpl:7102]:
```suggestion
              "expr": "rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval]) / rate(alertmanager_nflog_snapshot_duration_seconds_sum{instance=~\"$instance\"}[$__rate_interval])",
```

**Review by mmkay — COMMENTED** [review]
Couple more and we're good to go.

**Review by mmkay — APPROVED** [review]

**Pull request merged**